uefi/CHANGELOG.md

```diff
@@ -25,6 +25,9 @@
   `proto::device_path::text` to `proto::device_path`.
 - **Breaking:** `exit_boot_services` now consumes a `Option<MemoryType>` which
   defaults to the recommended value of `MemoryType::LOADER_DATA`.
+- **Breaking:** Removed duplication in `DevicePathHeader`. This change however
+  doesn't affect many users. Instead of public fields, there are public
```

Comment:
nit: I would take out "This change however doesn't affect many users." It's hard to predict how much code will be affected by a change, so better to stick with just what changed.